src/shogun/evaluation/SigmoidCalibration.h

```diff
@@ -139,6 +139,15 @@ namespace shogun
 		float64_t m_sigma;
 		/** Stopping criteria of search */
 		float64_t m_epsilon;
+	#ifndef SWIG
+	public:
+		static constexpr std::string_view kSigmoidAs = "m_sigmoid_as";
```

Comment:
if you touch this, I think we should remove the m_ prefix (which might require some refactoring down the line)

Reply:
Sounds good I will remove the m_prefix on the file

Reply:
could you do a quick `git grep m_sigmoid_as` etc to make sure no uses of the strings are missed? Thx :)

Reply:
should we only remove the m_prefix from "m_sigmoid_as" or from all of the names with the m_prefix on the file?

Reply:
all the names with the `m_` prefix please